english/robofriend/AIRobot.md

```diff
@@ -0,0 +1 @@
+You've been thinking of adopting one of those new AI robofriends. Yes, that's right. I know you've been lonely of late. [Here's an early present then.](AIattacks.md)
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.